tests/test_cell.py

```diff
@@ -14,6 +14,13 @@ def setUp(self):
         self.book = xlrd.open_workbook(from_this_dir('profiles.xls'), formatting_info=True)
         self.sheet = self.book.sheet_by_name('PROFILEDEF')
         
+    def test_empty_cell(self):
+        sheet = self.book.sheet_by_name('TRAVERSALCHAINAGE')
+        cell = sheet.cell(0, 0)
+        self.assertEqual(cell.ctype, xlrd.book.XL_CELL_EMPTY)
+        self.assertEqual(cell.value, '')

```

Comment:
Maybe I’m missing something, but isn’t comparing to an empty string ('') in Python 2 going to succeed for either kind of string? If you need to ensure it is a Unicode string maybe you should also do self.assertIsInstance(cell.value, unicode), or whatever the time machine equivalent is.